src/lib/components/ui/Modal/README.mdx

```diff
@@ -0,0 +1,332 @@
+              <p>
+                Nulla consequat massa quis enim. Donec pede justo, fringilla
+                vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus
+                ut, imperdiet a, venenatis vitae, justo.
+              </p>
+              <p>
+                Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean
+                commodo ligula eget dolor. Aenean massa.
+              </p>
+              <p>
+                Cum sociis natoque penatibus et magnis dis parturient montes,
+                nascetur ridiculus mus. Donec quam felis, ultricies nec,
+                pellentesque eu, pretium quis, sem.
+              </p>
+              <p>
+                Nulla consequat massa quis enim. Donec pede justo, fringilla
+                vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus
+                ut, imperdiet a, venenatis vitae, justo.
+              </p>
+              <p>
+                Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean
+                commodo ligula eget dolor. Aenean massa.
+              </p>
+              <p>
+                Cum sociis natoque penatibus et magnis dis parturient montes,
+                nascetur ridiculus mus. Donec quam felis, ultricies nec,
+                pellentesque eu, pretium quis, sem.
+              </p>
+              <p>
+                Nulla consequat massa quis enim. Donec pede justo, fringilla
+                vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus
+                ut, imperdiet a, venenatis vitae, justo.
+              </p>
+            </Modal>
+          )}
+        </div>
+      </>
+    );
+  }}
+</Playground>
+
+### Long Content and Autofocus
+
+👉 If you set the `scrollMode` to `modal`, you may want to turn `autoFocus` off
+to prevent the modal from scrolling to the end immediately after being opened.
+
+## Blocking Modals
+
+There are situations when you need to interrupt user's flow and lock entire UI
+to prevent interaction. That's where blocking modals may come handy.
+
+<Playground>
+  {() => {
+    const [modalOpen, setModalOpen] = React.useState(false);
+    return (
+      <>
+        <Button
+          clickHandler={() => setModalOpen(true)}
+          label="Launch blocking modal"
+          priority="outline"
+        />
+        <div>
+          {modalOpen && (
+            <Modal title="Update is in progress" size="auto">
+              <p>
+                <span role="img" aria-label="Warning">⚠️</span>
+                Please wait a few seconds until the firmware update is finished.
+                Don&apos;t turn off the device!
+              </p>
+              <Button
+                clickHandler={() => setModalOpen(false)}
+                label="Close the demo"
+                priority="flat"
+              />
+            </Modal>
+          )}
+        </div>
+      </>
+    );
+  }}
+</Playground>
+
+## API
+
+<Props of={ParsableModal} />
```

Comment:
Visual appearance of this table is not so good.

<img width="870" alt="image" src="https://user-images.githubusercontent.com/10213870/94276183-3202e480-ff48-11ea-81eb-410216aaff09.png">

Reply:
I'm aware of it but there is no quick way to fix that now. I will address it in an issue, we will probably turn this into a table anyway.